src/Sql/Sql/help/Az.Sql.md

```diff
@@ -167,9 +173,6 @@ Gets Advanced Data Security policy of a managed instance.
 ### [Get-AzSqlInstanceDatabase](Get-AzSqlInstanceDatabase.md)
 Returns information about Azure SQL Managed Instance database.
 
-### [Get-AzSqlInstanceDatabaseBackupShortTermRetentionPolicy](Get-AzSqlInstanceDatabaseBackupShortTermRetentionPolicy.md)
-Gets a backup short term retention policy.
-
```

Comment:
Why was this removed?

Reply:
I had a merge conflict where I added this line and it did not exist in the main repo. I guess it was removed by someone else, so I removed it too. This is not one of my cmdlets, so I will not add something owned by someone else